fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -73,6 +73,7 @@
 import static org.fcrepo.kernel.api.RdfLexicon.HAS_PRIMARY_TYPE;
 import static org.fcrepo.kernel.api.RdfLexicon.INBOUND_REFERENCES;
 import static org.fcrepo.kernel.api.RdfLexicon.INDIRECT_CONTAINER;
+import static org.fcrepo.kernel.api.RdfLexicon.JCR_NAMESPACE;
```

Comment:
I'm not saying you're wrong here, but if we are serous about people eventually picking up the `kernel-api`, impling it, and then using the HTTP modules with that impl, we will eventually want to remove this kind of JCR-related constant from the API entirely (even from our own namespaces) so this kind of testing is a little weird.